File: Windows/MagneticNote-WinForm/Model/Note.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MagneticNote.Model
{
    public class Note
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string Content { get; set; }
        public string CreateDate { get; set; }
        public string UpdateDate { get; set; }
        public int NoteBookId { get; set; }
    }

    public class NoteValue
    {
        public Note Note { get; set; }
    }

    public class NoteValues
    {
        public IList<Note> NoteList { get; set; }
    }
}

File: Android/app/src/main/java/date/slightcold/magneticnote/model/Note.java
package date.slightcold.magneticnote.model;

/**
 * Created by 29749 on 2017-12-13.
 */

public class Note {
    private int id;
    private String title;
    private String content;
    private String CreateDate;
    private String UpdateDate;
    private int NoteBookId;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getContent() {
        return content;
    }

    public void setContent(String content) {
        this.content = content;
    }

    public String getCreateDate() {
        return CreateDate;
    }

    public void setCreateDate(String createDate) {
        CreateDate = createDate;
    }

    public String getUpdateDate() {
        return UpdateDate;
    }

    public void setUpdateDate(String updateDate) {
        UpdateDate = updateDate;
    }

    public int getNoteBookId() {
        return NoteBookId;
    }

    public void setNoteBookId(int noteBookId) {
        NoteBookId = noteBookId;
    }

    @Override
    public String toString() {
        return this.title;
    }
}
